### PR TITLE
Updates for github pages 2021.12

### PR DESCRIPTION
**Pull request by bdmoore1**
Update for Github pages, the public version of the documentation.
Changes can be viewed in HTML format here: [https://bdmoore1.github.io/oneCCL/index.html
](https://bdmoore1.github.io/oneCCL/index.html
)
All of these changes were a part of the 2021.12 release.

### CODE REVIEW

**Review by nikitaxgusev — APPROVED** [review]
looks ok to me.

**Review by ksenyako — APPROVED** [review]
LGTM

File: _sources/introduction/installation.rst
Comment (maciekpac) [inline _sources/introduction/installation.rst:97]: not sure how it slipped through, I commented it at least twice and I saw a version that was ok; the link should lead to:
```suggestion
- `oneCCL Get Started Guide <https://www.intel.com/content/www/us/en/docs/oneccl/get-started-guide/current/overview.html>`_
```
right now, the link leads to outdated version of the website (to 2021.11, but the current release is 2021.12).

**Review by maciekpac — COMMENTED** [review]
One of the links that I already commented about in the past is outdated. Would be best to fix that.

Besides, the changes looks good.

**Comment by maciekpac** [comment]
@nikitaxgusev @ksenyako have you noticed the outdated link ? I'm not blocking the PR and not "requesting changes" per se, but I'm not sure if you noticed that and are ok with it.

**Comment by nikitaxgusev** [comment]
@maciekpac thought that it is aligned with @bdmoore1, lets see what he answers

**Comment by bdmoore1** [comment]
@maciekpac I used the 2021.12-gold branch to create the gh pages, and that link is still out of date in the release branch. https://github.com/intel-innersource/libraries.performance.communication.oneccl/blob/release/ccl_2021.12-gold/doc/rst/source/introduction/installation.rst
Master is up to date, so it will be fixed in the next release branch.
I accepted the commit. It looks like I have the ability to merge. Is that something an approver should do?

**Comment by nikitaxgusev** [comment]
@maciekpac review, and I will merge it, if it looks ok.

**Review by maciekpac — APPROVED** [review]

**Pull request merged**
